### PR TITLE
Add Thai translation

### PR DESCRIPTION
**Pull request by ssuntro**
Hello @basvodde,

My name is Anne from Thailand. i would like to offer Thai translation of Scrum Primer. And now the progress of Thai translation(th.yml) is 95% complete compare to English one(en.yml). But one problem i got is i cannot find the way to start website locally to cross check the change before move this PR from draft one to "READY for review". Can you guide me any npm  or any command i can use to start the env locally?

Cheers,
Anne

### CODE REVIEW

File: primer_source_files/overview_translations/scrum_japanese/Collected Fonts/Hiragino Kaku Gothic Pro/ヒラギノ角ゴ Pro W3.otf
Comment (ssuntro) [inline primer_source_files/overview_translations/scrum_japanese/Collected Fonts/Hiragino Kaku Gothic Pro/ヒラギノ角ゴ Pro W3.otf:1]: will undo this change before move the PR to "ready for review" status.

File: primer_source_files/overview_translations/scrum_japanese/Collected Fonts/Hiragino Kaku Gothic Pro/ヒラギノ角ゴ Pro W6.otf
Comment (ssuntro) [inline primer_source_files/overview_translations/scrum_japanese/Collected Fonts/Hiragino Kaku Gothic Pro/ヒラギノ角ゴ Pro W6.otf:1]: this is unrelated change. this change will be undo before move the PR to "ready for review" status.

**Comment by basvodde** [comment]
It should be with rake run_rackup_daemon.

Just rake will run all tests.

**Comment by ssuntro** [comment]
Update 

Currently, translation in th.yml file is complete reference from en.yml's content. So pls feel to review and validate changes  before merge the PR.


Anyways, there are 3 remaining action items which are
1. add keyword `scrum_primer_in_thai` in en.yml file
2. generate 3 documents - th_Scrum Primer v2.0.docx, th_scrumprimer20.pdf, and th_scrumprimer20_small.pdf
3. add href of `t.home.scrum_primer_in_thai` in home.erb file


Due to `rake run_rackup_daemon` is not working in my machine and i failed to start server locally so 
need to ask someone to carry over these 3 items later on.

Screen Shot 2566-04-25 at 20 51 20

File: primer_source_files/overview_translations/scrum_japanese/Collected Fonts/Hiragino Kaku Gothic Pro/ヒラギノ角ゴ Pro W3.otf
Comment (ssuntro) [reply primer_source_files/overview_translations/scrum_japanese/Collected Fonts/Hiragino Kaku Gothic Pro/ヒラギノ角ゴ Pro W3.otf:1]: this file is existing for 10 years now in master branch in origin repo and has no change present in my machine but for some reason the change of this file is spotted by this PR.

File: primer_source_files/overview_translations/scrum_japanese/Collected Fonts/Hiragino Kaku Gothic Pro/ヒラギノ角ゴ Pro W6.otf
Comment (ssuntro) [reply primer_source_files/overview_translations/scrum_japanese/Collected Fonts/Hiragino Kaku Gothic Pro/ヒラギノ角ゴ Pro W6.otf:1]: this file is existing for 10 years now in master branch in origin repo and has no change present in my machine but for some reason the change of this file is spotted by this PR.

**Comment by basvodde** [comment]
The error is because the link checker is not installed. Did you installed all the gems ?

**Review by yimmee123 — APPROVED** [review]
Thk

**Comment by basvodde** [comment]
Thanks
